contracts/staking/test/migration.ts

```diff
@@ -380,26 +316,6 @@ blockchainTests('Migration tests', env => {
             );
             expect(tx).to.revertWith(expectedError);
         });
-        it('reverts if wethAssetProxy is 0', async () => {
```

Comment:
I think we should still have these, only in the constructor for the deployment constants mixin.

Reply:
Added them!